### PR TITLE
fix(command): more detailed error message when version cannot be found

### PR DESCRIPTION
**Pull request by amorscher**
When running a lerna command and no version is present in lerna.json configuration following error message is printed:
```sh
lerna ERR! ENOLERNA `lerna.json` does not exist, have you run `lerna init`?
``` 
As described in (https://github.com/lerna/lerna/issues/3311) this is not completely correct.
## Description
Addtional check is added during command validation to prove a version is available in lerna.json file. In addition the check for having a lerna configuration (lerna.json) available is adapted to really check for existance of the config file. This is achieved by setting an additional flag configNotFound in the project object during lerna.json retrival.

## Motivation and Context
Improve error messages for invalid lerna.json to support user finding configuration problems.
The issue is described here (https://github.com/lerna/lerna/issues/3311).

## How Has This Been Tested?
Added a unit test.
Manually built a lerna version.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (change that has absolutely no effect on users)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by amorscher** [comment]
@JamesHenry any chance that the fix will make it  to main?

Thx for having a look!

File: core/project/index.js [hunk 1]
```diff
@@ -66,6 +66,7 @@ class Project {
             // No need to distinguish between missing and empty,
```
Comment (JamesHenry) [inline core/project/index.js:66]: Obviously I can't speak to the intent behind this comment originally, but your change does make it seem a little nonsensical, given you are adding an `isEmpty` right below a comment saying "no need to distinguish between missing and empty".

We could probably just amend the `this.configNotFound` assignment to check for `Object.keys(loaded.config ?? {}).length === 0`?

**Review by JamesHenry — CHANGES_REQUESTED** [review]
.

File: core/project/index.js (hunk 1 above)
Comment (amorscher) [reply core/project/index.js]: You are right that is a little bit weird. Just changed that and used your code suggestion.

**Review by JamesHenry — APPROVED** [review]

**Comment by JamesHenry** [comment]
Thanks @amorscher!

**Pull request merged**